src/SFA.DAS.ApplyService.Web/Views/Application/Pages/Controls/_Date.cshtml

```diff
@@ -14,6 +14,10 @@
     }
 }
 
+@* 
+    when there are multiple input elements with the same name the form variable on post back is (name = 'value1,value2,value3') this can be used to separate the components of the answer,
+    in the date component case since a ',' cannot be entered into any input element, this is a safe separator which can be used to separate the single answer value into 3 input elements.
+*@
```

Comment:
You can remove this comment.

Reply:
That was part of some other changes I rolled back